pnpm-lock.yaml

```diff
@@ -1461,6 +1461,9 @@ importers:
       '@votingworks/grout':
         specifier: workspace:*
         version: link:../../../libs/grout
+      '@votingworks/types':
+        specifier: workspace:*
+        version: link:../../../libs/types
```

Comment:
This large `pnpm-lock.yaml` diff seems to be undoing what I _think_ were accidental `pnpm-lock.yaml` changes introduced by https://github.com/votingworks/vxsuite/pull/3355

Will post in #vxsuite-prodeng later to make sure we're all on the same version of pnpm etc.